salt/apps/ocw/nextgen_build_publish.sls

```diff
@@ -0,0 +1,31 @@
+{% set ocw_next = salt.pillar.get('ocw-next') %}
+
+run_ocw_to_hugo:
+  cmd.run:
+    # Installing ocw-to-hugo without the `-g' switch leaves us without an
+    # `ocw-to-hugo' executable, so we use the path to `index.js'
+    - name: node src/bin/index.js -i /home/ocw/open-learning-course-data -o /home/ocw/hugo-course-publisher/site/content --strips3 --staticPrefix /coursemedia
+    # `cwd' is specified because it drops a log file here.
+    - cwd: /home/ocw/ocw-to-hugo
+    - runas: ocw
+
+run_hugo_course_publisher:
+  cmd.run:
+    - name: npm run build
+    # `cwd' is specified because it drops a log file here.
+    - cwd: /home/ocw/hugo-course-publisher
+    - runas: ocw
+    - env:
+      - NODE_OPTIONS: '-max_old_space_size=8192'
+    - require:
+      - cmd: run_ocw_to_hugo
```

Comment:
It looks like this state file is missing the step to handle the course static content?

Reply:
There's no more action needed, because the static course files (PDFs, images) are served from the raw data bucket. Fastly takes any path starting with `/coursemedia` and serves it from that bucket, instead of the website bucket.

[This line](https://github.com/mitodl/salt-ops/blob/b114d093febfd3d60abf0852d324d7174a724118/salt/apps/ocw/nextgen_build_publish.sls#L7) executes ocw-to-hugo with the options necessary to have it replace S3 bucket URLs with URLs that start with this `/coursemedia` path prefix.